File: VehiculosF2X.WebApi/VehiculosF2X.Dominio/Entidades/Recaudos.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace VehiculosF2X.Dominio.Entidades
{
    public class Recaudos
    {
        [Key]
        public int IdRecaudos { get; set; }
        public string Estacion { get; set; }
        public string Sentido { get; set; }
        public int Hora { get; set; }
        public string Categoria { get; set; }
        public int Cantidad { get; set; }
        public int ValorTabulado { get; set; }
        public DateTime Fecha { get; set; }
    }
}

File: recaudos-f2x/src/app/models/recaudos.ts
export class Recaudos {
    idRecaudos: number;
    estacion:  string;
    sentido: string;
    hora: number;
    categoria: string;
    cantidad: number; 
    valorTabulado: number;
    fecha: string;

    constructor(
        idRecaudos: number,
        estacion:  string,
        sentido: string,
        hora: number,
        categoria: string,
        cantidad: number, 
        valorTabulado: number,
        fecha: string) {
        this.idRecaudos = idRecaudos;
        this.estacion = estacion;
        this.sentido = sentido;
        this.hora = hora;
        this.categoria = categoria;
        this.cantidad = cantidad;
        this.valorTabulado = valorTabulado;
        this.fecha = fecha;
    }
}